language/stdlib/modules/LibraConfig.move

```diff
@@ -59,7 +59,7 @@ module LibraConfig {
         let signer_address = Signer::address_of(account);
         Transaction::assert(
             ::exists<ModifyConfigCapability<Config>>(signer_address)
-             || signer_address == Association::root_address(),
+             || signer_address == Association::root_address() || signer_address == Association::treasury_compliance_account(),
```

Comment:
Maybe we should do `Association::is_association(signer_address)` (which covers both root and TC) here?